docs/platforms/javascript/guides/angular/features/component-tracking.mdx

```diff
@@ -184,8 +184,7 @@ User `TraceDirective` if you only want to track components in certain instances
 
 ### Track Angular Bootstrapping
 
-You can add your own custom spans by attaching them to the currently active transaction using the `getActiveTransaction`
-helper. For example, you can track the duration of the Angular bootstrapping process to find out how long your app takes to bootstrap on your users' devices:
+You can add your own custom spans using `startSpan()`. For example, you can track the duration of the Angular bootstrapping process to find out how long your app takes to bootstrap on your users' devices:
```

Comment:
@Lms24 also noticed these here while going over this one more time and adjusted this, please sanity check if that all sounds reasonable for angular :D

Reply:
Just verified the snippet and it works 👍 However, we need to replace `description` with `name`